specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/locks.json

```diff
@@ -0,0 +1,1144 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ManagementLockClient",
+    "version": "2020-05-01",
+    "description": "Azure resources can be locked to prevent other users in your organization from deleting or modifying resources."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "AuthorizationOperations_List",
+        "description": "Lists all of the available Microsoft.Authorization REST API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
```

Comment:
Need to add schema for error. You can reference common-types https://github.com/Azure/azure-rest-api-specs/blob/002ad52075e725708deabf842a736c98751b8566/specification/common-types/resource-management/v2/types.json#L322